app/code/community/Kirchbergerknorr/GoogleTagManager/etc/system.xml

```diff
@@ -51,11 +51,20 @@
                             <show_in_website>1</show_in_website>
                             <show_in_store>0</show_in_store>
                         </id>
+                        <html_position translate="label">
+                            <label>Position in HTML</label>
+                            <comment>Use a HTML tag, e.g. "after_body_start" or "head" (without quotation marks)</comment>
+                            <frontend_type>text</frontend_type>
```

Comment:
It might be a good idea to use a dropdown instead to reduce input error propability.